spring-web-modules/spring-thymeleaf/src/main/java/com/baeldung/thymeleaf/config/WebApp.java

```diff
@@ -29,8 +31,7 @@ protected String[] getServletMappings() {
     }
 
     @Override
-    protected void customizeRegistration(final Dynamic registration) {
-        super.customizeRegistration(registration);
+    protected void customizeRegistration(ServletRegistration.Dynamic registration) {
```

Comment:
It seems we are overriding this method but just calling the parent inside. Can we remove this so we just simply inherit the one from parent?

Reply:
Done